### ISSUE TITLE
Add "beta tag" to TaxBrain

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/webapp-public#37]
We will be removing the admonition against publishing anything with TaxBrain results shortly, but I see no reason not to leave it in "beta" for the first part of its public life. 

Is the best/easiest way to accomplish this by adding a "beta" label to the taxbrain logo everywhere it appears on the home page, input page, and output page, and pdf output?

**Comment by talumbau** [comment]
I think it would involve editing these images and adding the word 'Beta' in some way that is aesthetically pleasing:

```
./static/images/logo-taxbrain.png
./static/images/taxbrain/logo-taxbrain-alt.png
./static/images/taxbrain/logo-taxbrain-bw.png
./static/images/taxbrain/logo-taxbrain-footer.png
./staticfiles/images/logo-taxbrain.png
./staticfiles/images/taxbrain/logo-taxbrain-alt.png
./staticfiles/images/taxbrain/logo-taxbrain-bw.png
./staticfiles/images/taxbrain/logo-taxbrain-footer.png
```

These files appear to be repeated in the repo for reasons that aren't clear to me. We could create new assets for each of the 4 PNG files with the same filename but `_beta.png`. Then change all references to be the `_beta` ones until such time that we wish to remove the "Beta".

We have graphic designers that could do this relatively quickly. It's not clear we would be able to do a 100% match on the fonts, but a good match on the color is easy and as long as folks didn't look too closely it would probably look OK. Let me know if you think this is a good way to go.

**Comment by MattHJensen** [comment]
> We have graphic designers that could do this relatively quickly. It's not clear we would be able to do a 100% match on the fonts, but a good match on the color is easy and as long as folks didn't look too closely it would probably look OK. Let me know if you think this is a good way to go.

This sounds good to me. I'm hoping for something quick and dirty... a beta tag ought to be a makeshift thing.

**Comment by talumbau** [comment]
@MattHJensen sounds like this would be a fast thing to do. Do you want the "Beta" in parentheses or brackets or anything like that?

-T.J.

**Comment by MattHJensen** [comment]
I'll defer to the designers (or to you if you have a preference).

**Comment by talumbau** [comment]
closed via #42
